### PR TITLE
@scale/vagrant file for @scale deployment

### PR DESCRIPTION
**Pull request by chenge3**
This PR is Vagrant example file for @scale deployment on vSphere and Libvirt provider.
Wiki Page - **Vagrant and Libvirt plugin for @scale deployment**
https://github.com/InfraSIM/infrasim-compute/wiki/Vagrant-and-Libvirt-plugin-for-@scale-deployment

Wiki Page - **Vagrant and vSphere plugin for @scale deployment**
https://github.com/InfraSIM/infrasim-compute/wiki/Vagrant-and-vSphere-plugin-for-@scale-deployment

### CODE REVIEW

**Comment by chenge3** [comment]
@turtle-fly @XiaowenJiang @WinnieXU1 @xiar @fub2 
Any comments please let me know. Thanks.

File: @scale/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Vagrant files for vagrant-libvirt and vagrant-vaphere
```
Comment (fub2) [inline @scale/README.md]: typo

**Comment by fub2** [comment]
wiki page is at infrasim-compute repo and this is targeting tool repo. How about incorporating wiki page into readme file you placed here into @scale folder?

**Comment by chenge3** [comment]
@fub2 Will do!

**Comment by JuneZhou19** [comment]
how to avoid conflict between default vagrant file and user's vagrant file? Add comments in readme?

**Comment by JuneZhou19** [comment]
Suggest to hint for user to create resource pool/datacenter or use existing?

**Comment by XiaowenJiang** [comment]
@JuneZhou19 Rename default one to vagrantfile.bk I think ....

**Comment by fub2** [comment]
use case for Vagrant: first step to create a folder and then either place a vagrantfile or create one from scratch (usually from vagrant add box, vagrant init, etc). Then it is time to run vagrant up. Seems it is not necessary to remind that there's conflicts.

**Comment by JuneZhou19** [comment]
@fub2 OK, then we can forget that if most people are well aware of that.

**Comment by JuneZhou19** [comment]
Datacenter name is required to specify when there are multiple datacenter. So suggest to improve the statement "vsphere.data_center_name = 'Data-center-Rack' #Optional, data center containing the computed resource."

**Comment by chenge3** [comment]
@JuneZhou19 @XiaowenJiang @turtle-fly Have a try~~. Comments are always welcomed.

**Comment by JuneZhou19** [comment]
For libvirt section, I tried on a fresh machine which don't have packer, but packer is needed in our steps. So can we have packer installation in the steps or mention about the precondition of the host?

**Comment by fub2** [comment]
Packer required? sound strange. Agree if it is true.

**Comment by JuneZhou19** [comment]
Suggest to change "vagrant box add **builds/**ubuntu-14.04.libvirt.box --name "ubuntu1404" ------to----->"vagrant box add **dir/to/**ubuntu-14.04.libvirt.box --name "ubuntu1404"

**Comment by JuneZhou19** [comment]
Suggest to change "vagrant box add builds/ubuntu-14.04.libvirt.box --name "ubuntu1404" ------to----->"vagrant box add dir/to/ubuntu-14.04.libvirt.box --name "ubuntu1404"

**Comment by fub2** [comment]
What this update is for? @JuneZhou19 . you can comment in review.

File: @scale/vagrant_libvirt/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+   Packer build templates found in [ chef/bento github repo.](https://github.com/chef/bento).
+    
+	$ git clone https://github.com/chef/bento
+
+	$ cd bento
+
+	$ sudo packer build -only qemu -var "headless=true" ubuntu-14.04-amd64.json
+
+	$ vagrant box add builds/ubuntu-14.04.libvirt.box --name "ubuntu1404"
```
Comment (JuneZhou19) [inline @scale/vagrant_libvirt/README.md]: It fails on execution of the command "vagrant box add builds/ubuntu-14.04.libvirt.box --name" at my side, not sure if the "builds/ubuntu-14.04.libvirt.box" meant for directory path to file or url? If directory to file, I would recommend to change it to dir/to/ubuntu-14.04.libvirt.box?

Comment (JuneZhou19) [inline @scale/vagrant_libvirt/README.md]: It's OK to improve command later. So I would give +1 now.

**Comment by JuneZhou19** [comment]
+1

**Pull request merged**